### PR TITLE
SAK-31820 Fix roster display of roster group

### PR DESCRIPTION
**Pull request by master-bob**
Roster groups where being shown regardless of what the permission
was. This was fixed.

### CODE REVIEW

File: roster2/src/handlebars/members.handlebars [hunk 1]
```diff
@@ -66,7 +66,7 @@
                 {{/if}}
             {{/if}}
 
-            {{#if hasGroups}}
+            {{#if ../viewGroup}}
```
Comment (ottenhoff) [inline roster2/src/handlebars/members.handlebars]: Can this if statement check for presence of groups and the permission?

Comment (master-bob) [inline roster2/src/handlebars/members.handlebars]: The if statement cannot handle conditionals.

If the members loaded has no groups associated with them, but there are groups associated with other unloaded members than the groups header will not be shown (this was present with the hasGroups condition).

**Comment by juanjmerono** [comment]
Please, check my comment in jira:
https://jira.sakaiproject.org/browse/SAK-31820?focusedCommentId=210065&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-210065

**Comment by ottenhoff** [comment]
Are we clear to proceed?

**Pull request merged**